web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx

```diff
@@ -20,48 +20,119 @@ import { Text, Box } from 'design';
 import { AWSIcon } from 'design/SVGIcon';
 
 import cfg from 'teleport/config';
-import { ToolTipNoPermBadge } from 'teleport/components/ToolTipNoPermBadge';
+import {
+  BadgeTitle,
+  ToolTipNoPermBadge,
+} from 'teleport/components/ToolTipNoPermBadge';
 import { IntegrationKind } from 'teleport/services/integrations';
 
+import { ToolTipBadge } from 'teleport/components/ToolTipBadge';
+
 import { IntegrationTile } from './common';
 
-// IntegrationTiles is plural but at the moment we only
-// support aws-oidc. Expecting this to grow.
 export function IntegrationTiles({
-  hasAccess = true,
+  hasIntegrationAccess = true,
+  hasExternalAuditStorage = true,
 }: {
-  hasAccess?: boolean;
+  hasIntegrationAccess?: boolean;
+  hasExternalAuditStorage?: boolean;
 }) {
+  // TODO(mcbattirola): isUsageBasedBilling is used here and in other
+  // parts of the app as synonym with Team product, but this
+  // will change in the future.
+  const isEnterprise = cfg.isEnterprise && !cfg.isUsageBasedBilling;
```

Comment:
are all current usage of `isUsageBasedBilling` mean team right?

Reply:
Yes, right now isUsageBasedBilling === `team` (so the `cfg.isEnterprise` is redundant, but I wanted to be explicit).